specification/_toc.yml

```diff
@@ -4,14 +4,15 @@
 format: jb-book
 root: intro
 parts:
-- caption: User Guide
-  chapters:
-  - file: guide
-  - file: legacy
-- caption: Specification
-  chapters:
-  - file: reference
-  - file: validator
-- caption: Examples and resources
-  chapters:
-  - file: model-zoo
+  - caption: User Guide
+    chapters:
+      - file: json_101
```

Comment:
This is fine for now, but I think when we get to the point of releasing it will make more sense in examples and resources, rather than as the most prominent thing in the site.

Reply:
yup I agree
I think we can put somewhere else and might jsut need something in the intro that goes: "if you are unfamiliar with JSON files, please see JSON 101 before reading further'